### ISSUE TITLE
Linux desktop integration troubleshooting

### ISSUE THREAD

**Issue by mobitar** [standardnotes/desktop#195]
User report:

> I get prompted everytime I run the .AppImage. I press Yes and nothing happens. Now I tried to run it in terminal and I've seen an error. The SN.AppImage couldn't create file in the ~/.local/share/applications. This folder didn't exist before. I created the folder and the problem was solved.

**Comment by charlec** [comment]
I run Fedora (Gnome 3 as standard) and don't have this problem. Probably need to see if this is distribution dependent?

**Comment by mobitar** [comment]
Yeah, results seem to be varied with the hundred flavors of Linux..
